modules/tooltip/react/stories/Tooltip.stories.mdx

```diff
@@ -0,0 +1,130 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+import {Tooltip} from '@workday/canvas-kit-react-tooltip';
+import {Specifications} from '@workday/canvas-kit-specifications';
+
+import {Default} from './examples/Default';
+import {CustomContent} from './examples/CustomContent';
+import {DescribeType} from './examples/DescribeType';
+import {Placements} from './examples/Placements';
+import {Ellipsis} from './examples/Ellipsis';
+import {LineClamp} from './examples/LineClamp';
+import {UseTooltip} from './examples/UseTooltip';
+
+<Meta title="Components/Popups/Tooltip/React" component={Tooltip} />
+
+# Canvas Kit React Tooltips
+
+A Tooltip component that renders information/text when the user hovers over an element. A tooltip is
+used to label or describe an element. By default, a tooltip will label an element. This is useful
+for IconButtons. A tooltip can also be used to describe additional information about an element
+
+[> Workday Design Reference](https://design.workday.com/components/popups/tooltips)
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+or
+
+```sh
+yarn add @workday/canvas-kit-react-tooltip
+```
+
+## Usage
+
+This component follows the
+[W3 Tooltip specification](https://www.w3.org/TR/wai-aria-practices/#tooltip). Tooltips are used to
+label icon buttons and provide additional context to elements.
+
+### Basic Example
+
+Here is a basic example of an `IconButton` using a tooltip to label the icon. This is labeling the
+button for both sighted users and for screen readers. A tooltip will provide an `aria-label` to
+child elements for the accessibility tree and a visual tooltip during mouse hover and focus events.
+Mouse over or focus on the x icon below. Click "Show code" to see the source code.
+
+<ExampleCodeBlock code={Default} />
+
+### Describing an Element
+
+The default mode for a tooltip is to label content via `aria-label`. If a tooltip is meant to
```

Comment:
I would consider removing the critical part of this messaging. I propose the following:
The default mode for a tooltip is to label content via `aria-label`. If a tooltip is meant to
provide ancillary information, the `type` can be set to `describe`. This will add
`aria-describedby` to the target element. This will allow screen reader users to hear the name of the control that is being focused and the ancillary tooltip information.

Reply:
Thanks! Will update.